documentation/docs/index.md

```diff
@@ -3,11 +3,12 @@ notoc: true
 icon: "/vespucci_logo_small.png"
 ---
 
-[<img src="badge-amazon-appstore.png" style="padding-top:30px;padding-left:20px;float:right">](http://www.amazon.com/Simon-Poole-vespucci/dp/B00P71C3AW)
-[<img src="badge-google-play.png" style="padding-top:30px;padding-left:20px;float:right">](https://play.google.com/store/apps/details?id=de.blau.android)
-
 # Welcome to Vespucci 
 
+[<img src="badge_f-droid.png" style="margin-top:30px;" height="150" width="502" alt="F-Droid">](https://f-droid.org/en/packages/de.blau.android/)
```

Comment:
Please remove this, we don't want to refer to a build that we have no control over.